ingest/validation/validate_metadata.py

```diff
@@ -915,8 +915,24 @@ def process_metadata_row(metadata, convention, line):
     row_info = dict(itertools.zip_longest(metadata_names, line))
     processed_row = {}
     for k, v in row_info.items():
-        # for metadata not in convention, no need to process
+        # for metadata not in convention, check numeric metadata for valid values
         if k not in convention["properties"].keys():
+            type_index = metadata.headers.index(k)
+            k_type = metadata.annot_types[type_index]
+            if k_type == "numeric":
+                k_numeric = type(v) == int or type(v) == float
+                if not k_numeric:
+                    # pandas coercion will stringify numbers
+                    try:
+                        float(v)
+                        msg = f"{k}: one or more values are not numeric"
```

Comment:
Could you clarify, at least in a code comment if not also the `msg`, a specific case or two that trigger this?

```suggestion
                        msg = f"{k}: one or more values are not numeric"
```

Reply:
Done! c91ca9b added an example in the comments and, hopefully, this updated message is more precise:
`one or more values in data column are not numeric`